### DISCUSSION TITLE
Unarchived type filter on repositories page

### DISCUSSION THREAD

**Discussion by farski** [community/community#52669] [Repositories]
### Select Topic Area

Product Feedback

### Body

On the repositories page ([e.g.](https://github.com/orgs/PRX/repositories)), there's an option to filter down to [Archived](https://github.com/orgs/PRX/repositories?q=&type=archived&language=&sort=) repositories.

It would be very helpful if there were also an option to only show repositories that are _not_ archived.

**Comment by opegamanalitarok** [comment]
GitHub does provide an option to filter repositories by their archived status. However, there is currently no built-in option to filter repositories by their unarchived status.

One workaround to achieve this is to use the search bar at the top of the repositories page and enter archived:false. This will filter the repositories to show only those that are not archived.

To do this, follow these steps:

1. Go to the repositories page on GitHub.
2. Click on the search bar at the top of the page.
3. Enter archived:false in the search bar and press enter.
4. GitHub will filter the repositories to show only those that are not archived.

Note that this method will only work if you are viewing a list of repositories that includes archived repositories. If you are already viewing a list of unarchived repositories, then this search query will not have any effect.

**Comment by randytoo** [comment]
This doesn't work if you are looking at your team repos.  The filter seems to look at the archived:false as a name and not an attribute there.

**Comment by rasa** [comment]
Please consider adding a "Not archived" option to the "Type" dropdown on a user's repositories page aka https://github.com/rasa?tab=repositories
